wolox-training/nh-android
# Profile frontend

## Profile Frontend

### Summary
Implemented frontend of profile.
User can see profile information, but can't edit.
When the profile or cover images are tapped, they are shown in fullscreen.

**Cards:** 
* Card: [11] https://trello.com/c/ip1iWYKQ/11-profile

### Test Plan
1) Log in with an existing user if not already logged in, for example using mail "mail@test.com" and password "password", swipe left for profile, user information should appear, as shown in Screenshot 1.
2) Tap on profile picture to open it in fullscreen, as shown in Screenshots 2.
3) Tap on cover image to open it in fullscreen, as shown in Screenshot 3.

### Screenshots

#### Screenshot 1
![profile](https://cloud.githubusercontent.com/assets/14595211/10495800/1f76adf6-7294-11e5-8242-3b217f786e4e.png)

#### Screenshot 2
![profile-big-avatar](https://cloud.githubusercontent.com/assets/14595211/10495801/20ff22ac-7294-11e5-9e29-1ad4b71605ff.png)

#### Screenshot 3
![profile-big-cover](https://cloud.githubusercontent.com/assets/14595211/10495802/21e2c494-7294-11e5-889f-3ba54f76fad6.png)
